<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>315</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Tahoma</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(58, 14, 61);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>15</y>
      <width>470</width>
      <height>291</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:15px;
background-color: rgb(50, 12, 49);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>441</width>
      <height>241</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(56, 4, 65);
border-radius:15px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="c">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>140</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(45, 11, 47);</string>
    </property>
    <property name="text">
     <string>Current Value -</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_currentvalue">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>140</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(45, 11, 47);</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>100</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(45, 11, 47);
</string>
    </property>
    <property name="text">
     <string>Application -</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_name">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>100</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(45, 11, 47);

</string>
    </property>
    <property name="text">
     <string>NotFound</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>120</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(45, 11, 47);</string>
    </property>
    <property name="text">
     <string>Process id -</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_processid">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>120</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(45, 11, 47);</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>211</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(45, 11, 47);
border-radius:15px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(45, 11, 47);


</string>
    </property>
    <property name="text">
     <string>Information :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>180</y>
      <width>113</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(58, 14, 61);
</string>
    </property>
    <property name="placeholderText">
     <string>Enter Offset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>230</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>verify</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>40</y>
      <width>171</width>
      <height>221</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(45, 11, 47);
border-radius:15px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>50</y>
      <width>151</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(45, 11, 47);
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Tahoma'; font-size:7pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Enter the name and offset of what you want to read/write to of the target application and then click the verify button to read/write to its memory&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>150</y>
      <width>113</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(58, 14, 61);
</string>
    </property>
    <property name="placeholderText">
     <string>Enter Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_changevalue">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>200</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
     </font>
    </property>
    <property name="placeholderText">
     <string>Change Value</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>230</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>change</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(45, 11, 47);
</string>
    </property>
    <property name="text">
     <string>Memory Status :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_idle">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>170</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(45, 11, 47);
</string>
    </property>
    <property name="text">
     <string>Idle</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_idle_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>230</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(45, 11, 47);
</string>
    </property>
    <property name="text">
     <string>Idle</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>230</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(45, 11, 47);
</string>
    </property>
    <property name="text">
     <string>Status -</string>
    </property>
   </widget>
   <widget class="QLabel" name="c_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>35</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(56, 4, 65);
</string>
    </property>
    <property name="text">
     <string>Madeby- Amogh Raina</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>280</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:5px;
background-color: rgb(50, 12, 49);</string>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <item>
     <property name="text">
      <string>purple theme</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>dark theme</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>280</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(50, 12, 49);


</string>
    </property>
    <property name="text">
     <string>themes :</string>
    </property>
   </widget>
   <zorder>label_8</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>c</zorder>
   <zorder>label_5</zorder>
   <zorder>label_name</zorder>
   <zorder>label_7</zorder>
   <zorder>label_currentvalue</zorder>
   <zorder>label_processid</zorder>
   <zorder>label_4</zorder>
   <zorder>label_6</zorder>
   <zorder>lineEdit</zorder>
   <zorder>pushButton</zorder>
   <zorder>textEdit</zorder>
   <zorder>lineEdit_2</zorder>
   <zorder>lineEdit_changevalue</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>label_idle</zorder>
   <zorder>label_idle_2</zorder>
   <zorder>label_10</zorder>
   <zorder>c_2</zorder>
   <zorder>comboBox</zorder>
   <zorder>label_9</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
